tests/io/netcdf/test_to_netcdf.py

```diff
@@ -0,0 +1,146 @@
+#! /usr/bin/env python
```

Comment:
I think a test for RadialModelGrid.to_netcdf() is missing...

And/or equivalent of the last test in test_from_netcdf.py

Reply:
There are some tests for `RasterModelGrid.to_netcdf()` but I should add some more/better ones.